### DISCUSSION TITLE
Variáveis de Ambiente - Chavear Configurações

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2801#74] [Perguntas e Respostas] [answered]
O chaveamento de configurações de ambiente (dotnet run --environment=development, production, etc.) serve, por exemplo:

Para que em ambiente de produção o dotnet run use um arquivo que define acesso ao banco de dados _X_ com a senha _123_. 

Já em ambiente de desenvolvimento ele vai usar o arquivo que define acesso ao banco de dados _localhost:8080/base_ com a senha _admin_.

Essa variação do dotnet run serve pra coisas desse tipo? Se puder dar exemplos mais "globais"/uma visão mais "macro" eu agradeço.

**Comment by andrebaltieri** [comment, marked as answer]
@PolarisTDM sim, é exatamente para isto... você pode criar seus próprios nomes se quiser, os mais comuns são `dev`, `qa`, `prod`
